interview-questions.md

```diff
@@ -7,62 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: (Even though Austin went through this in class I just wanted to complete it) Conditional statements are used to make decisions in your code and then make something happen based on the result. They use booleans to determine their decision. You can use if statements and the ternary operator to achieve these!
 
-  Researched answer:
+  Researched answer: Conditionals can determine the end or beginning of a certain process. Also just found out you can also use !== for strict inequality along with nesting conditionals.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is the software on your local machine that allows you to interact(e.g. push/pull) with Github which is the remote. We use git and Github on a daily basis to update the code base.
 
-  Researched answer:
+  Researched answer: git is a version control tool used to log your changes on files and applications and push them to the remote repository on Github
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: Logical and relational operators return boolean values along with the equality operators. Super useful for useful functions that ask questions to both the user and the machine.
 
+  Researched answer: Undefined is the result if your value isn't returned and null as intentional absence of a value(considered falsy)
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: an index is the position of a element within another data type. Think like a number within an array, the index also starts at 0. A value could be a number, boolean, or other primitive data types. The difference between them is each value has an assigned index based on its position in the code.
 
-  Researched answer:
+  Researched answer: An index can be viewed as a list assigning itself to individual data types. Values tell you the data type that is contained in an index
```

Comment:
Nice, I think there can be more here that is uncovered but this is a good start.